camera_data.py

```diff
@@ -0,0 +1,19 @@
+import cv2
+import math
+import random
+
+
+class CameraData:
+    def __init__(self, camera_tilt):
+        self.H_FIELD_OF_VIEW = 68.37
```

Comment:
Have we verified these values with the cameras we are using this year?

Reply:
No. How would we verify these values?